iosxr_test.py

```diff
@@ -72,25 +76,30 @@ def bringup_vagrant():
     except OSError:
         pass
 
-    print('Cleaning up ssh keys')
-    run('ssh-keygen -R [localhost]:2222')
-    run('ssh-keygen -R [localhost]:2223')
-
-    print("Bringing up '%s'..." % input_box)
+    logger.debug("Bringing up '%s'..." % input_box)
 
-    run('vagrant init XRv64-test')  # Single node for now, in future could bring up two nodes and do more testing
-    run('vagrant box add --name XRv64-test %s --force' % input_box)
-    run('vagrant up')
+    run(['vagrant', 'init', 'XRv64-test'])  # Single node for now, in future could bring up two nodes and do more testing
+    run(['vagrant', 'box', 'add', '--name', 'XRv64-test', input_box, '--force'])
+    output = run(['vagrant', 'up'])
+    print(output)
 
     # Find the correct port to connect to
     port = subprocess.check_output('vagrant port --guest 57722', shell=True)
+    logger.debug('Connecting to port %s' % port)
 
     try:
-        s = pexpect.pxssh.pxssh()
+        s = pxssh.pxssh(options={
+            "StrictHostKeyChecking": "no",
+            "UserKnownHostsFile": "/dev/null"})
         s.login(hostname, username, password, terminal_type, linux_prompt, login_timeout, port)
+        logger.debug('Sucessfully brought up VM and logged in')
+        s.logout()
     except pxssh.ExceptionPxssh, e:
-        print("pxssh failed on login")
-        print(str(e))
+        logger.error("pxssh failed on login")
+        logger.error(e)
+
+    logger.debug('Waiting 30 seconds...')
+    time.sleep(30)
```

Comment:
I had an issue when I did. Removing it now things are much more stable and retesting.